src/main/java/com/Crowdsourcing/messages/CrowdsourcingMessages.java

```diff
@@ -143,6 +143,10 @@ public class CrowdsourcingMessages
 	private static final int CHARM_OF_PREPARATION = 12100;
 	private static final int CHARM_OF_BOOST = 12104;
 
+  // Hallowed Sepulchre coffins
+  private static final String SEPULCHRE_FAILURE = "You have been poisoned.";
+  private static final String SEPULCHRE_SUCCESS = "You push the coffin lid aside.";
```

Comment:
This is going to fire on the grand coffin right? Can you fail that one? Something to keep in mind when we query the data since that could bias results. Worth adding a comment on these messages for these cases so whoever queries the data knows.